packages/nc-gui-v2/components/cell/TimePicker.vue

```diff
@@ -22,13 +22,20 @@ const localState = $computed({
     if (!modelValue) {
       return undefined
     }
+    let dateTime = dayjs(modelValue)
 
-    if (!dayjs(modelValue).isValid()) {
+    if (!dateTime.isValid()) {
+      dateTime = dayjs(modelValue, 'HH:mm:ss')
+    }
+    if (!dateTime.isValid()) {
+      dateTime = dayjs(`1999-01-01 ${modelValue}`)
+    }
+    if (!dateTime.isValid()) {
       isTimeInvalid = true
       return undefined
     }
 
-    return dayjs(modelValue)
+    return dateTime
```

Comment:
What about `dateTime.format('HH:mm:ss')` ?

Reply:
@wingkwong Ant date picker expects a dayjs object and takes a date format too. So this is taken care of.

Reply:
Oh ok then.